### PR TITLE
chore: Add issue section to contributing guide

### LINKED ISSUE #24: Add an Issue Template to Contributing Guide
There is no format on how to create an issue in the contributing guide and I believe that just as making a PR and commiting good, clear messages are important, creating a good issue that is clear and straight forward is just as important.

### PR DESCRIPTION
**Pull request by d-a-ve**
Closes #24

### CODE REVIEW

**Comment by DominusKelvin** [comment]
Awesome @d-a-ve Ii will have Geekiedj review this.

**Comment by d-a-ve** [comment]
Alright, sounds good @DominusKelvin

**Comment by geekiedj** [comment]
Hi @d-a-ve, thank you for your contribution. Instead of using [chore], [feat], or [fix] prefixes in the issue title, it's recommended to use labels for categorization. These prefixes are commonly used in commit messages or PR titles, not when creating an issue.

**Comment by d-a-ve** [comment]
Okay, so labels such as [bug], [doc], etc. Or what kind of labels do you have in mind @geekiedj

**Comment by geekiedj** [comment]
> Okay, so labels such as [bug], [doc], etc. Or what kind of labels do you have in mind @geekiedj

yeah labels such as "good first issues, enhancement, etc. You can find them on the right of this page or better still, you could check the issues tab to understand what I'm talking about

**Comment by d-a-ve** [comment]
> > Okay, so labels such as [bug], [doc], etc. Or what kind of labels do you have in mind @geekiedj
> 
> yeah labels such as "good first issues, enhancement, etc. You can find them on the right of this page or better still, you could check the issues tab to understand what I'm talking about

I have seen them; bug, enhancement, good-first-issue, wont-fix etc. So these should be added to the issue, alright. Will the person be the one to select the issue label that fits or that will be done by the maintainer? @DominusKelvin @geekiedj

**Comment by geekiedj** [comment]
> > > Okay, so labels such as [bug], [doc], etc. Or what kind of labels do you have in mind @geekiedj
> > 
> > 
> > yeah labels such as "good first issues, enhancement, etc. You can find them on the right of this page or better still, you could check the issues tab to understand what I'm talking about
> 
> I have seen them; bug, enhancement, good-first-issue, wont-fix etc. So these should be added to the issue, alright. Will the person be the one to select the issue label that fits or that will be done by the maintainer? @DominusKelvin @geekiedj

I assume you're trying to add a `create issue` in the contributing. md guideline right? if that's correct, then all you have to do will be to ask(i.e, include that step in the contributing guide) whoever might be interested in creating an issue in the repo (IF, the issue doesn't exist) to include labels to the issue that way other contributors will understand what the issue intends to solve as well as a descriptive title and comment of course.

Does that make sense? @d-a-ve

**Comment by d-a-ve** [comment]
Yes this is great, I understand. So the person picks the labels that fits the issue. 

I will make a pull request later in the day for the changes.

**Comment by d-a-ve** [comment]
@geekiedj What do you think? It seems the person that creates an issue cannot assign the labels themselves...unless I do not know how to do it

**Comment by geekiedj** [comment]
> @geekiedj What do you think? It seems the person that creates an issue cannot assign the labels themselves...unless I do not know how to do it

Oh! Just checked now, that's true. It has to do with some permission things. If that be the case, you could take that option out and just make a PR.

**Comment by d-a-ve** [comment]
Okay, I didn't add it to the last commit I sent, I was waiting for your feedback. Can you check the issue section I created in the last commit I made? So we can be sure it is okay before merging.

**Comment by d-a-ve** [comment]
Okay, I didn't add it to the last commit I sent, I was waiting for your feedback. Can you check the issue section I created in the last commit I made? So we can be sure it is okay before merging.

**Comment by d-a-ve** [comment]
Okay, I didn't add it to the last commit I sent, I was waiting for your feedback. Can you check the issue section I created in the last commit I made? So we can be sure it is okay before merging.

**Comment by d-a-ve** [comment]
Okay, I didn't add it to the last commit I sent, I was waiting for your feedback. Can you check the issue section I created in the last commit I made? So we can be sure it is okay before merging.

**Comment by geekiedj** [comment]
> Okay, I didn't add it to the last commit I sent, I was waiting for your feedback. Can you check the issue section I created in the last commit I made? So we can be sure it is okay before merging.
> 
> Here is the commit [5ef7466](https://github.com/sailscastshq/boring-stack/commit/5ef7466c21adffacbe40123460492d5ac01a980b)

I thought I replied already.Apologies.  I think this looks good. Let's see what @DominusKelvin  thinks?

**Comment by d-a-ve** [comment]
Alright. @DominusKelvin What do you think?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -29,6 +29,17 @@ To contribute to The Boring JavScript Stack, follow these steps:
 
 ---
 
+## Creating an Issue
+Before **creating** an issue, please follow these steps:
+
+1. Search existing issues before creating a new issue (has someone created an issue already).
```
Comment (DominusKelvin) [inline CONTRIBUTING.md:35]: Cool, can we rephrase it to do this @d-a-ve  👇🏾
Before creating a new issue, it is advisable to search for any existing issues that may have already been created.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -29,6 +29,17 @@ To contribute to The Boring JavScript Stack, follow these steps:
...
+2. If it doesn't exist, create an issue giving as much context and information as possible in the description. 
+> Images and videos can be added to the description to provide more information.
+3. Use an appropriate format for the title:  `[bug]`, `[documentation]`, `[help wanted]`, or `[good first issue]` etc, followed by a concise title description. For example, an issue that relates to The Boring Tech Stack documentation could have a title like `documentation: Add issue section to Contributing guidelines`.
```
Comment (DominusKelvin) [inline CONTRIBUTING.md:38]: I think the example didn't reference the guide, so can it be rephrased and also for prefix, let's go with bug:, docs:, help wanted: etc

Hence you example will look like docs: Add issue section to Contributing guidelines

FYI, the angle bracket syntax is intended for PRs

**Review by DominusKelvin — CHANGES_REQUESTED** [review]
@d-a-ve this looks good. I pointed out a couple of things. Once they are fixed, I'll proceed to merge this. Thanks :)

**Comment by DominusKelvin** [comment]
@d-a-ve thanks for working on this, I left a couple of suggestions for you :)

File: CONTRIBUTING.md (hunk 2 above)
Comment (d-a-ve) [reply CONTRIBUTING.md:38]: I didn't know the angle brackets are for PRs, alright.

I will make the necessary changes and let you know

**Comment by d-a-ve** [comment]
@DominusKelvin I have made the changes. How is it now?

**Comment by d-a-ve** [comment]
Hi @DominusKelvin please have you been able to check the change i did with the new commit?
